_config.yml

```diff
@@ -2,4 +2,4 @@ theme: jekyll-theme-minimal
 title: Geek Source
 description: An online shopping experience providing a number of real-world challenges front-end developers face daily as professional developers.
 logo: https://wesdollar.github.io/geek-source/assets/logo.png
-markdown: CommonMarkGhPages
+markdown: GFM
```

Comment:
This may break the formatting on Richard's collapsible content. If you have Jekyll setup to run locally, please verify. Otherwise, I'll verify and address accordingly.